scripts/Dockerfile.devel

```diff
@@ -1,10 +1,17 @@
 FROM ceph/ceph:v15
 
-ENV GOPATH=/go \
-	GO111MODULE=on
+
+RUN \
+ mkdir -p /go && \
+ curl https://storage.googleapis.com/golang/go1.13.9.linux-amd64.tar.gz | tar xvzf - -C /go --strip-components=1
+
+ENV GOROOT=/go
```

Comment:
@humblec can you reduce all env to one ENV

Reply:
Done.. updated the patchset